script.xbmc.boblight/default.py

```diff
@@ -0,0 +1,277 @@
+# -*- coding: utf-8 -*- 
+'''
+    Boblight for Kodi
+    Copyright (C) 2012 Team XBMC
```

Comment:
please adjust the license headers, this addon is no longer provided by Team Kodi